Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -1195,10 +1199,18 @@ void GetHitTestResultsLegacy(std::vector<HitResult>& filteredResults, xr::HitTes
         m_impl->sessionImpl.DeleteAnchor(anchor);
     }
 
+    System::Session::Frame::SceneObject& System::Session::Frame::GetSceneObjectByID(System::Session::Frame::SceneObject::Identifier) const {
+        throw std::runtime_error("not implemented");
```

Comment:
are these to be implemented later as well?

Reply:
yes, this logic exists for ARKit, but does not currently exist for ARCore

Reply:
nit: may be add a TODO comment if that helps

Reply:
I've added https://github.com/BabylonJS/BabylonNative/issues/542

I'll add a TODO calling out this issue if i need to submit additional changes/rerun ci.